source/module_lr/potentials/xc_kernel.cpp

```diff
@@ -37,7 +37,7 @@ void LR::KernelXC::f_xc_libxc(const int& nspin, const double& omega, const doubl
 
     // -----------------------------------------------------------------------------------
     // for GGA
-    const bool is_gga = std::any_of(funcs.begin(), funcs.end(), [](const xc_func_type& f) { f.info->family == XC_FAMILY_GGA || f.info->family == XC_FAMILY_HYB_GGA; });
+    const bool is_gga = std::any_of(funcs.begin(), funcs.end(), [](const xc_func_type& f)->bool { return f.info->family == XC_FAMILY_GGA || f.info->family == XC_FAMILY_HYB_GGA; });
```

Comment:
good practice, you now use one line to get whether "it" is gga functional, so `->bool` is a type hint?

Reply:
Oh yes, `->bool` is not necessary here because the lhs. is not declated as "auto".